src/App.js

```diff
@@ -361,7 +361,6 @@ const AppStack = () => {
     >
       <Stack.Navigator
         screenOptions={{
-          presentation: 'modal',
```

Comment:
Since the iOS modals exhibition have changed, this property is no longer desired.